### PR TITLE
Styled the number facts card

### PR DESCRIPTION
**Pull request by shonali2600**
## Related Issue

Designed a card for number facts showing on the home page.

Closes: #241 

#### Describe the changes you've made

I've added shadow, hover effect, and changed background color to make it look good.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         | 
| :---------------------: | :------------------------: |
| then
 
|         Updated           |
 | before |

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -301,13 +303,24 @@ h1, h2, h3, h4, h5, h6 {
     
 }
 #numberfact{
-    font-family: mainfont;
+    font-family: 'Berkshire Swash', cursive;
```
Comment (shrey27tri01) [inline css/style.css:306]: The font you added, "Berkshire Swash", is not much user-friendly. Could you add another more user-friendly and readable font, like Roboto or Open Sans? Or you can even leave the font as it was previously.

**Review by shrey27tri01 — CHANGES_REQUESTED** [review]
I just had some inline reviews. Please have a look. 
You just need to change the font. Other than that, the box and box-shadow you added are looking awesome!

**Comment by shonali2600** [comment]
@shrey27tri01 sir, I changed the font to its previous style. Thanks for giving feedback.

**Review by sairish2001 — APPROVED** [review]
Well Done!
Thanks for your contribution

**Review by shrey27tri01 — APPROVED** [review]
Great job!

**Pull request merged**